db/schema.rb

```diff
@@ -10,10 +10,16 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_04_14_131424) do
+ActiveRecord::Schema[7.0].define(version: 2022_04_21_162919) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "local_authorities", force: :cascade do |t|
```

Comment:
Given this data is pretty much static - I wonder if it's better to read from a config file?

Reply:
The thinking was that it might not be static, and being database backed would give us flexibility in the future.

Reply:
👍